assets/javascripts/kitten/components/tooltips/tooltip-icon.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export default class TooltipIcon extends React.Component {
+  render() {
+    return (
+      <button className="k-ButtonIcon
```

Comment:
On peut éventuellement ajouter `aria-describedby={ this.props.id }` pour relier le bouton à la tooltip.